src/controllers/localCache.controller.ts

```diff
@@ -75,10 +72,11 @@ export const getLocalCache: Handler = async (req, res) => {
             message: messageData,
             data: responseData,
         });
-    } catch (error) {
-        return res.status(200).send({
+    } catch (error: any) {
+        const code = error instanceof AxiosError ? error.response?.status || 500 : 500;
+        return res.status(code).send({
             error: true,
-            message: error,
+            message: error.message,
```

Comment:
Are u sure this is working with every catched error? we use this for the circulary dependency for handling just one case, try to be more generic with error handling